src/js/api/core-shim.js

```diff
@@ -95,7 +96,12 @@ Object.assign(CoreShim.prototype, {
         if (!model.get('backgroundLoading')) {
             mediaPool = SharedMediaPool(mediaPool.getPrimedElement(), mediaPool);
         }
-        mediaPool.prime();
```

Comment:
Note that all video tags are "primed" as they are created. So this call is no longer necessary after creating the pool.